content/en/docs/concepts/workloads/controllers/_index.md

```diff
@@ -3,3 +3,41 @@ title: "Workload Resources"
 weight: 20
 ---
 
+Kubernetes provides several built-in APIs for declarative management of your
+{{< glossary_tooltip text="workloads" term_id="workload" >}}
+and the components of those workloads.
+
+You use Kubernetes API to create {{< glossary_tooltip text="object" term_id="object" >}}
+(HTTP resources) that represent these higher level abstractions, and then let the
```

Comment:
"higher level" in this context may confuse new users, i.e. why you call them "higher", comparing to what?. Maybe "abstractions" is sufficient.

Reply:
I reworded.